pkg/apis/elasticsearch/v1alpha1/types.go

```diff
@@ -61,6 +61,8 @@ type ElasticsearchNode struct {
 	Resources    v1.ResourceRequirements  `json:"resources"`
 	NodeSelector map[string]string        `json:"nodeSelector,omitempty"`
 	Storage      ElasticsearchStorageSpec `json:"storage"`
+	// this will be provided and populated by the operator
+	GenUUID *string `json:"genUUID,omitempty"`
```

Comment:
formatting

Reply:
the comment prevented `gofmt` from matching it to the line above... let me play around with comment location